workloads/network.go

```diff
@@ -1,380 +1,224 @@
+// Package workloads includes workloads types (vm, zdb, qsfs, public IP, gateway name, gateway fqdn, disk)
```

Comment:
can we please make the file wgnetwork, as it's more relevant to wireguard. znet is also a good name given it's the primitive in zos